models/employee.py

```diff
@@ -46,6 +47,10 @@ def get_current_employee(cls):
         user_email = user.email()
         employee = cls.query(cls.user == user, cls.terminated == False).get()  # noqa
         if employee is None:
+            # if you change the GAE domain settings, the user objects change
+            # and the above query fails. this query will work for that
+            employee = cls.get_employee_by_email(user_email)
+        if employee is None:
```

Comment:
Yeah I suppose it's OK as is.  I just did a bit of a double take when I read it.